README.md

```diff
@@ -83,6 +83,7 @@ This project would not exist without the contributions of many students in the D
 Alex Baluta,
 Alexander Biggs,
 Kelly Bell,
+Ching Chang,
```

Comment:
This is showing changes that have now been merged in from your other PR. You should update this branch by doing a merge from `upstream master`, and these changes will no longer appear. 👍